src/domain/User.java

```diff
@@ -51,6 +52,18 @@ public String getLastName() {
         return lastName;
     }
 
+    /**
+     * Updates the user's first name and last name and returns the user
+     * @param firstname - the new first name
+     * @param lastname - the new last name
+     * @return User
+     */
+    public User update(String firstname, String lastname) {
+        this.firstName = firstname;
+        this.lastName = lastname;
+        return this;
```

Comment:
update(String firstname, String lastname, String password) 
this.frirstName - firstName
this.lastNAme = lastName
 setPassword(password)
return this